docs/public-networks/reference/engine-api/index.md

```diff
@@ -17,6 +17,51 @@ See the [Ethereum Engine API specification](https://github.com/ethereum/executio
 
 ## Methods
 
+### `engine_exchangeCapabilities`
+
+Exchanges a list of supported Engine API methods between the consensus client and Besu.
+
+#### Parameters
+
+`remoteCapabilities`: *array* of *strings* - Array of names of the Engine API methods the consensus
+client supports
+
+#### Returns
+
+`localCapabilities`: *array* of *strings* - Array of names of the Engine API methods Besu supports
+
+!!! example
+
+    === "curl HTTP"
+
+        ```bash
+        curl -X POST --data '{"jsonrpc":"2.0","method":"engine_exchangeCapabilities","params":[["engine_exchangeTransitionConfigurationV1","engine_forkchoiceUpdatedV1","engine_getPayloadBodiesByHash","engine_getPayloadBodiesByRangeV1","engine_getPayloadV1","engine_newPayloadV1"]],"id":67}' http://127.0.0.1:8550
+        ```
+
+    === "wscat WS"
+
+        ```bash
+        {"jsonrpc":"2.0","method":"engine_exchangeCapabilities","params":[["engine_exchangeTransitionConfigurationV1","engine_forkchoiceUpdatedV1","engine_getPayloadBodiesByHash","engine_getPayloadBodiesByRangeV1","engine_getPayloadV1","engine_newPayloadV1"]],"id":67}
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+            "jsonrpc": "2.0",
+            "id": 67,
+            "result": [
+              "engine_exchangeTransitionConfigurationV1",
+              "engine_forkchoiceUpdatedV1",
+              "engine_getPayloadBodiesByHashV1",
+              "engine_getPayloadBodiesByRangeV1",
+              "engine_getPayloadV1",
+              "engine_newPayloadV1"
```

Comment:
Might be good to also add engine_forkchoiceUpdatedV2, engine_newPayloadV2, engine_getPayloadV2
since the point of this RPC is to show when the next version is supported (generally we'll have two supported versions at any one time).

Reply:
e.g.

```
{
    "jsonrpc": "2.0",
    "id": 68,
    "result": [
        "engine_getPayloadV1",
        "engine_getPayloadV2",
        "engine_executePayloadV1",
        "engine_newPayloadV1",
        "engine_newPayloadV2",
        "engine_forkchoiceUpdatedV1",
        "engine_forkchoiceUpdatedV2",
        "engine_exchangeTransitionConfigurationV1",
        "engine_getPayloadBodiesByHashV1",
        "engine_getPayloadBodiesByRangeV1"
    ]
}
```